tests/vlan/test_vlan.py

```diff
@@ -134,8 +134,11 @@ def setup_vlan(ptfadapter, duthosts, rand_one_dut_hostname, ptfhost, vlan_ports_
         logger.info("Add members to Vlans")
         for vlan_port in vlan_ports_list:
             for permit_vlanid in vlan_port['permit_vlanid'].keys():
+                is_untagged_member = True if vlan_port['pvid'] == permit_vlanid else False
+                if vlan_port['dev'].startswith("Ethernet") and is_untagged_member is True:
+                    duthost.command('config vlan member del 1000 {port}'.format(port=vlan_port['dev']))
```

Comment:
Why line 130 didn't remove the port from vlan 1000?

Reply:
Thanks. After I sync the solution of PR#2973, the port can be removed from vlan 1000.